src/main/resources/twilio-go/partial_serialization.mustache

```diff
@@ -24,13 +24,13 @@ if params != nil && params.PathAccountSid != nil {
 {{#hasOptionalParams}}
 {{#optionalParams}}
 {{#vendorExtensions.x-is-body-param}}
+    body := []byte{}
     if params != nil && params.{{paramName}} != nil {
-        v, err := json.Marshal(*params.{{paramName}})
+        b, err := json.Marshal(*params.{{paramName}})
         if err != nil {
             return nil, err
         }
-        urlEncodedData := url.QueryEscape(string(v))
-        data.Set("{{baseName}}", urlEncodedData)
+        body = b
```

Comment:
No, before adding json support for request payload, the payload was sent as key value pairs, so we need to set the base name and its value in data object, so in the end data variable will have lots of key value pairs containing value for each variable

Now since its json, there will be single json object (which can be nested also) as payload. So we just need to set a variable with that json body and pass to Post method